src/cacheout/cache.py

```diff
@@ -24,7 +24,7 @@
 UNSET = object()
 
 
-class EvictionCause(Enum):
+class RemovalCause(Enum):
     """
     An enum to represent the cause for the eviction of a cache entry.
```

Comment:
nit: Can replace "eviction" with "removal".
```suggestion
    An enum to represent the cause for the removal of a cache entry.
```